dolphin/common/config.py

```diff
@@ -88,6 +88,17 @@
 
 CONF.register_opts(global_opts)
 
+redis_opts = [
+    cfg.StrOpt('db_name', default=''),
+    cfg.StrOpt('user', default=''),
+    cfg.StrOpt('password', secret=True),
+    cfg.StrOpt('redis_ip', default='127.0.0.1'),
```

Comment:
default ip and port also can be added to constants.py

Reply:
Yes, and I found that in dolphin/alert_manager, there is a constants.py, and I think it may be better to put all the constants of every module into one constants.py. So I want to move all the constants in dolphin/alert_manager/constants.py into dolphin/common/constants.py. How do you think about it?